### PR TITLE
Resolve issues #167 #171

### PR DESCRIPTION
**Pull request by trafalgarandre**

### CODE REVIEW

**Comment by kush1509** [comment]
Entering path for profile picture which doesn't exist doesn't give error. Try fixing that.

File: docs/DeveloperGuide.adoc [hunk 1]
```diff
@@ -22,15 +22,21 @@ Let's get started!
 
 == Setting up
 
-=== Prerequisites
-Given below are the required prerequisites for developing contactHeRo.
+Before you start to develop contactHeRo, let's us help you to set up.
```
Comment (kush1509) [inline docs/DeveloperGuide.adoc]: Can change to:
Before you start contributing to contactHeRo, let us ...

File: docs/DeveloperGuide.adoc [hunk 2]
```diff
@@ -55,12 +61,15 @@ You can follow the steps below to set up contactHeRo in your computer.
 This will generate all resources required by the application and tests.
 
 === Verifying the setup
+You can double check whether your setup is right or not by:
```
Comment (kush1509) [inline docs/DeveloperGuide.adoc]: .... following the steps below.

File: docs/DeveloperGuide.adoc [hunk 3]
```diff
@@ -92,15 +101,20 @@ Optionally, you can set up AppVeyor as a second CI (see <<UsingAppVeyor#, UsingA
 [NOTE]
 Having both Travis and AppVeyor ensures your App works on both Unix-based platforms and Windows-based platforms (Travis is Unix-based and AppVeyor is Windows-based)
 
-==== Getting started with coding
+=== Starting your journey.
```
Comment (kush1509) [inline docs/DeveloperGuide.adoc]: This sounds wrong. Let the original thing be haha.

File: docs/DeveloperGuide.adoc [hunk 4]
```diff
@@ -92,15 +101,20 @@ Optionally, you can set up AppVeyor as a second CI (see <<UsingAppVeyor#, UsingA
...
 
-When you are ready to start coding,
+Now you are ready to start coding. +
+Here are some tips to get started with developing contactHeRo:
```
Comment (kush1509) [inline docs/DeveloperGuide.adoc]: change developing -> contributing

File: docs/DeveloperGuide.adoc [hunk 5]
```diff
@@ -92,15 +101,20 @@ Optionally, you can set up AppVeyor as a second CI (see <<UsingAppVeyor#, UsingA
...
 
 1. Get some sense of the overall design by reading <<Design-Architecture>>.
 2. Take a look at <<GetStartedProgramming>>.
 
+Thanks for joining us in developing contactHeRo! Have fun!
```
Comment (kush1509) [inline docs/DeveloperGuide.adoc]: same as above

**Comment by trafalgarandre** [comment]
#173 #167, #171, #177, #175, #187, #201, #185, #199

**Pull request merged**